src/util.rs

```diff
@@ -134,10 +120,151 @@ pub fn run_input_output<C>(mut command: C, input: Option<Vec<u8>>)
              }))
 }
 
-#[test]
-fn test_os_str_bytes() {
-    // Just very basic sanity checks in case anyone changes the underlying
-    // representation of OsStr on Windows.
-    assert_eq!(os_str_bytes(OsStr::new("hello")), b"hello");
-    assert_eq!(os_str_bytes(OsStr::new("你好")), "你好".as_bytes());
+pub trait OsStrExt {
+    fn starts_with(&self, s: &str) -> bool;
+    fn split_prefix(&self, s: &str) -> Option<OsString>;
```

Comment:
Definitely! We on the libs team have often wanted some form of string-like API added to `OsStr` as well, we just never figured out the best way to do so... For now less efficient implementations :(